src/temporal/config.ts

```diff
@@ -12,7 +12,7 @@ interface ServerOptions {
 }
 
 export const TEMPORAL_ADDRESS =
-  process.env.TEMPORAL_ADDRESS || 'host.docker.internal:7233';
+  process.env.TEMPORAL_ADDRESS || 'localhost:7233';
```

Comment:
is this accidental change?

Reply:
No, this should be good until we have the workers spooled within Tilt.

Reply:
👌